Makefile.am

```diff
@@ -24,11 +24,13 @@ SUBDIRS = \
   mc \
   $(NEUTRINORDPDIR) \
   libxrdp \
+  librfxcodec \
   xrdp \
   sesman \
   keygen \
   docs \
   instfiles \
   genkeymap \
   xrdpapi \
+  xorgxrdp \
```

Comment:
Likewise, we need to decide if we want xorgxrdp to be part of the tree. I believe most users would want to use the `/usr` prefix for xorgxrdp to make it work, yet they make want to use `/usr/local` or other prefix for xrdp. I would rather have xorgxrdp as a completely separate package.